bin/download_schemas.sh

```diff
@@ -0,0 +1,95 @@
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-cmn-json-schema.json"
+      exit 1
+    fi
+    #Remove key '$schema' of the common file because it would overwrite the same key in the main file
+    jq 'del(."$schema")' umm-cmn-json-schema-temp.json > umm-cmn-json-schema.json
+    #Merge and create js file
+    echo "$(echo "const ummCSchema =")" "$(jq -s '.[0] * .[1]' umm-c-json-schema.json umm-cmn-json-schema.json)" > ummCSchemaRaw.js
+    echo "$(echo "export default ummCSchema")" >> ummCSchemaRaw.js
+    #Replace pointers to the common file
+    sed 's/umm-cmn-json-schema.json#/#/g' ummCSchemaRaw.js > ummCSchema.js
+}
+
+ummS() {
+    echo "Downloading ummS"
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummS' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/service/v${schema_version}/umm-s-json-schema.json > umm-s-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-s-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-s-json-schema-temp.json > umm-s-json-schema.json
+    #Create js file
+    echo "$(echo "const ummSSchema =")" "$(jq '.' umm-s-json-schema.json)" > ummSSchema.js
+    echo "$(echo "export default ummSSchema")" >> ummSSchema.js
+}
+
+ummV() {
+    echo "Downloading ummV"
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummV' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/variable/v${schema_version}/umm-var-json-schema.json > umm-var-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-var-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-var-json-schema-temp.json > umm-var-json-schema.json
+    #Create js file
+    echo "$(echo "const ummVarSchema =")" "$(jq '.' umm-var-json-schema.json)" > ummVarSchema.js
+    echo "$(echo "export default ummVarSchema")" >> ummVarSchema.js
+}
+
+ummT() {
+    echo "Downloading ummT"
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummT' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/tool/v${schema_version}/umm-t-json-schema.json > umm-t-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-t-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-t-json-schema-temp.json > umm-t-json-schema.json
```

Comment:
I wonder if we can clean up these -temp files when done?

Reply:
Added cleanup to remove temp files

Reply:
I just ran locally, there is still some files leftover in bin directory can these be cleaned up too?

Reply:
Yes, they all now removed